packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -290,18 +291,45 @@ export const createBundle = async ({
                 } = await getProjectPkg()
                 const extendsTemplate = 'node_modules/' + ccExtensibility.extends
 
+                const pwaKitDeps = [
+                    '@salesforce/pwa-kit-dev',
+                    '@salesforce/pwa-kit-runtime',
+                    '@salesforce/pwa-kit-react-sdk'
+                ]
+
                 let cc_overrides: string[] = []
-                if (ccExtensibility.overridesDir) {
+                const packageVersions: {[key: string]: string} = {}
+
+                if (ccExtensibility.extends) {
+                    const extendsTemplateVersion =
+                        dependencies[ccExtensibility.extends] ||
+                        devDependencies[ccExtensibility.extends]
                     const overrides_files = await walkDir(
                         ccExtensibility.overridesDir,
                         ccExtensibility.overridesDir
                     )
                     cc_overrides = Array.from(overrides_files).filter((item) =>
                         existsSync(path.join(extendsTemplate, item))
                     )
+
+                    // If the project is using template extensbility, we must use 'npm info' to find the versions of pwa-kit packages
+                    try {
+                        const packageInfo = JSON.parse(
+                            execSync(
+                                `npm info ${ccExtensibility.extends}@${extendsTemplateVersion} --json`
+                            ).toString()
+                        )
```

Comment:
Using `npm info` removes the need to care about the structure of `node_modules`, but it introduces a network request. Using ``require.resolve(`${ccExtensibility.extends}/package.json`)`` also solves the problem, but doesn't require a network request. But both of these solutions assume that the PWA Kit packages are immediate dependencies of the `ccExtensibility.extends` package, when that's not guaranteed to be the case. Using the output of `npm ls` or `package-lock.json` will enable searching the full dependency tree, rather than just the first layer (plus both of them use the local file system, rather than network requests).